### PR TITLE
refactor(mod-tile): simplified deeply nested functions

### PR DESCRIPTION
**Pull request by CptSchnitz**

### CODE REVIEW

File: images/mod-tile/start.py [hunk 1]
```diff
@@ -29,7 +29,15 @@
...
 
-def extract_positivie_integer_value(text, key):
+def cast_string_to_int(value):
+    try:
+        return int(value)
+    except ValueError:
+        return None
+
+# positive thinking
+# simplify function
```
Comment (syncush) [inline images/mod-tile/start.py]: remove

File: images/mod-tile/start.py [hunk 2]
```diff
@@ -128,6 +129,31 @@ def update_rendered_state_file(rendered_state_file_path, sequence_number):
...
+    sequence_number = get_sequence_number_from_file(file_path,'sequenceNumber')
+    if not sequence_number:
+        raise LookupError(f'"sequenceNumber=" is not present in file')
+    return sequence_number
+
+def get_rendered_sequence_number(file_path):
+    if not path.exists(file_path):
+        log.info('initializing rendering state file')
+        # with open(file_path, 'w') as rendered_file:
+        #     rendered_file.write('lastRendered=1')
```
Comment (syncush) [inline images/mod-tile/start.py]: remove

**Review by syncush — CHANGES_REQUESTED** [review]

File: images/mod-tile/start.py [hunk 3]
```diff
@@ -237,12 +242,12 @@ def handle_command_successful_complete():
 
 if __name__ == '__main__':
     log_file_extention = '.log'
-    mod_tile_name = 'mod-tile'
+    rendered_name = 'rendered'
     base_log_path = path.join('/var/log', app_name)
     service_logs_path = path.join(base_log_path, app_name + log_file_extention)
-    mod_tile_log_path = path.join(base_log_path, mod_tile_name + log_file_extention)
+    mod_tile_log_path = path.join(base_log_path, rendered_name + log_file_extention)
```
Comment (vitaligi) [inline images/mod-tile/start.py]: should `mod_tile_log_path` be renamed to `renderd_log_path`?

File: images/mod-tile/start.py [hunk 4]
```diff
@@ -128,6 +127,29 @@ def update_rendered_state_file(rendered_state_file_path, sequence_number):
...
+    if not sequence_number:
+        raise LookupError(f'"sequenceNumber=" is not present in file')
+    return sequence_number
+
+def get_rendered_sequence_number(file_path):
+    if not path.exists(file_path):
+        log.info('initializing rendering state file')
+        rendered_file = open(file_path, 'w')
+        rendered_file.write('lastRendered=1')
+        rendered_file.close()
```
Comment (vitaligi) [inline images/mod-tile/start.py]:
```python3
with open(file_path, 'w') as rendered_file:
    rendered_file.write('lastRendered=1')
```
is a better way handling file writes

**Review by syncush — APPROVED** [review]

**Pull request merged**